### ISSUE TITLE
Clothing no longer hides other clothing items and hair when it should.

### ISSUE THREAD

**Issue by PetMudstone** [unitystation/unitystation#6908]
### Environment
- Game Version: Staging 21070122
- OS Version: Windows 10

### Detailed description of bug
Clothes that should hide other clothing items or hair no longer do. For example, in the below images the skirt and syndicate magboots should be hidden by the advanced hardsuit but still render, albeit under said hardsuit. Meanwhile, helmets that should hide hair do not but only for clients. From the host's point of view hair is properly hidden.
As seen by the host:
![image](https://user-images.githubusercontent.com/38266309/124241782-ee3d9c80-dad0-11eb-8ac1-ebee74689d2d.png)
As seen by the client:
![image](https://user-images.githubusercontent.com/38266309/124242198-568c7e00-dad1-11eb-8711-ba02f82021e3.png)

### Steps to reproduce
1. Wear clothing that should hide other clothing, such as a firesuit.
2. Wear clothing that ought to clip with the clothing from step 1, such as a skirt or clown shoes.
3. Notice the clothing from step 2 still render and clip with the clothing from step 1.

**Comment by PetMudstone** [comment]
Related to #6711.

**Comment by Fogapod** [comment]
Same for wings/tail

**Comment by PerfectTangent** [comment]
related to #4473

**Comment by PerfectTangent** [comment]
I haven't seen this issue in a while. Can anyone reproduce?

**Comment by CdrCross** [comment]
Can verify that this is still a problem during the Playtest on the 29th of Jan 2022. In my case, it was the Detective's Fedora and CT cap that hair clipped right through.

**Comment by PerfectTangent** [comment]
yeah, I actually saw it during the playtest as well specifically with underwear.
